plop-templates/storybook.test.stories.tsx.hbs

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import type { {{pascalCase name}} } from '@amsterdam/design-system-react/src'
+import type { Meta, StoryObj } from '@storybook/react-vite'
+
+import { default as {{camelCase name}}Meta } from './{{pascalCase name}}.stories'
+
+const meta = {
+  ...{{camelCase name}}Meta,
+  title: '{{titleCase name}}',
+} satisfies Meta<typeof {{pascalCase name}}>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Test: Story = {
+  // NOTE: When adding new variants, ensure they are included in the test story's render function for visual regression testing.
```

Comment:
This template is only used when you set up a new component. When you've done that, this comment will probably get deleted. So maybe we should adapt the time / phrasing of the comment to that point in time?
Something like:

```suggestion
  // NOTE: Add all variants of the component to this story for visual regression testing.
```